yetanotherco/aligned_layer
# fix: periodic updates for batcher page

## Motivation

Details like age and badge weren't updating. Fixes #1646.

* Extract batch age to a field in the dict so LiveView knows when it changed
* Broadcast an event on the block update task so views know to update assigns

## Type of change


## Testing

1. Deploy a devnet and the explorer.
2. Send a few proofs.
3. Stop sending proofs.
4. Go to the ex batches page. You should see the age column updating about once a minute.
5. Kill the operator.
6. Send a few more proofs.
7. Stop sending proofs.
8. After a while (5 minutes by default) the pending proofs should start turning red, with the badge signaling "unverified".
9. Verify also for a given batch and for the frontpage.

## Checklist

  - [Link to the open issue addressing it]()
